### PR TITLE
Feature 7/ FE Skeleton Ready to test

### PR DESCRIPTION
**Pull request by dimitrov93**

### CODE REVIEW

**Comment by dimitrov93** [comment]
Ticket: https://trello.com/c/UHC4rKeg/7-fe-create-fe-skeleton

**Comment by dimitrov93** [comment]
Branch to Test PR - https://github.com/JewelleryManagement/jewellery-management-ui/pull/2

**Review by aleks247 — APPROVED** [review]
I tested it and it works perfectly !

**Review by aleks247 — COMMENTED** [review]
I tested it and it works perfectly !

**Review by aleks247 — APPROVED** [review]
I tested it and it works perfectly !

File: README.md
Comment (aleks247) [inline README.md:1]: You described the steps well

File: vue.config.js
Comment (aleks247) [inline vue.config.js:1]: That's good

**Review by aleks247 — APPROVED** [review]
I tested it and it works perfectly !

**Review by aleks247 — APPROVED** [review]
I tested it and it works perfectly !

**Pull request merged**